app.R

```diff
@@ -1,4 +1,345 @@
-# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Thea Wenxin
-# date: 2022-03-01
+# author: Christopher Alexander, Jennifer Hoang, Michelle Wang, Wenxin Xiang
+# date: 2022-03-15
 
-pass
+library(dash)
+library(dashHtmlComponents)
+library(dashBootstrapComponents)
```

Comment:
Not sure why , but I am not able to install this package in my local . I only need `dashHtmlComponents` to run the code .